Add buttons and features to tannerweb

viskey98 opened the pull request:
I have implemented the following:

**Features:**
- Mushorg logo in the index-page.
- Snares button in the index-page.
- Home and Doc and button on top right corner of every page

**Stats:**
- Latest session ​info in index-page.
- Number of snares info in index-page.
- Changed ‘attack_types’ to ‘attacks’ in session info. This will now show the count of every attack types.
- New identity ‘admin’ for the info ‘Possible Owners’ of session-stats
- Moved location to filters

afeena commented on tanner/web/templates/base.html:
```diff
@@ -1,5 +1,28 @@
+			</a>
+			|
+			<a href="/static/manual.pdf" style="text-decoration:none;" >
+				Commands-manual
```
I suggest you to make a filters manual instead of commands, where you can list all available filters. Or we can keep in the docs and even delete this commands-manual page

afeena commented:
Looks good :)

viskey98 commented:
@afeena As the tannerweb doc already contained the filter details, I just added a 6th filter 'location' into the docs. And I have removed the Commands-manual button from the tannerweb UI. Please look into it :)
